test/cypress/integration/getting_started_spec.ts

```diff
@@ -20,6 +20,8 @@ const goToNextStep = () => cy.contains('button', 'Next').click();
 const goToFinishStep = () => cy.contains('button', 'Finish').click();
 const closeModal = () => cy.contains('button', 'Close').click();
 
+const waitOptionsForPluginsUpload = { responseTimeout: 5 * 60 * 1000 };
```

Comment:
Minor:

We could add a type annotation here for better discoverability. Too bad `WaitOptions` is not exported from Cypress types :worried: This is the best solution I could come up with, but it could be overly verbose:

```suggestion
const waitOptionsForPluginsUpload: Parameters<typeof cy.wait>[1] = { responseTimeout: 5 * 60 * 1000 };
```

Reply:
Applied your suggestion.